CanalTP/SamEcoreUserManagerBundle
# Adding created_at column in user table

# Description
This PR adds created_at column in user table in order to expose them to datalayer

## Reference
Reference link: BOT-590

## Pull Request type

This is new feature

## How to test

Here are the following steps to test this pull request:

- from terminal at project root launch ```./app/console claroline:migration:upgrade CanalTPSamEcoreUserManagerBundle```
- a new column ```usr_created_at``` should be created in ```public.t_user_usr``` table
- if you create a new user from nmm application, current date should be used as user creation date (check the value of column ```usr_created_at``` in  ```public.t_user_usr``` table)
